scriptworker/constants.py

```diff
@@ -1,628 +0,0 @@
-#!/usr/bin/env python
```

Comment:
I don't love that a bunch of these files look like they've been deleted and recreated, rather than renamed.
If your rename patch preserves that history, then we should look at *not* squashing and merging this PR, so we keep history across the rename.

Reply:
Yeah, that's not ideal. The original commit does show them as renames: https://github.com/mozilla-releng/scriptworker/pull/413/commits/78f823d22c16ea960e32e95382b5c0fefb6e8333#diff-607a0befd08dc071b4d4863adc55c118

I agree that preserving the history makes sense. Given that, do you want me to do some rebasing to clean up the more recent batch of commits a bit after we're done with review passes?

Reply:
My main concern is preserving history, so I'm glad the first commit does that. You're welcome to do any rebasing that makes sense. I'm not going to be too picky about it as long as we keep that first commit. Thanks!